packages/whats-that-gerber/index.js

```diff
@@ -84,7 +84,7 @@ var layerTypes = [
     name: {
       en: 'board outline'
     },
-    match: /(Edge.Cuts)|(\.((dim$)|(mil$)|(gm[l\d]$)|(gko$)|(fab$)))|(\.boardoutline\.ger$)|(\.outline\.gbr$)|(mechanical \d+\.txt$)/i
+    match: /(Edge.Cuts)|(\.((dim$)|(mil$)|(gm[l\d]{1,2}$)|(gko$)|(fab$)))|(\.boardoutline\.ger$)|(\.outline\.gbr$)|(mechanical \d+\.txt$)/i
```

Comment:
This would also match stuff like `.gmll` and `.gm2l` , which I don't think we intend. Maybe this instead?

```js
/...(gm(?:l|\d{1,2})$).../
```

Reply:
Good spot, thanks. What's the reason for `?:`? I think we can just do `(gm(l|\d{1,2})$)`.

Reply:
Oh sorry, should've clarified. (?:...) means a non-capturing group, as opposed to (...). Capturing is regex thing that i, frankly, did not understand when I first wrote this code. Pretty much every (...) here should be (?:...) because these regexes are only ever called with regex.test, which means capturing doesn't matter

Reply:
Yeah, good point, better to keep it consistent